tools/cdf-lineage-exporter/cdap_lineage_export/main.py

```diff
@@ -198,7 +198,7 @@ def export_all_cdap_lineage_to_gcs(project: str,
             print(
                 f"no streams lineage found for time period: {start_ts} to {end_ts}"
             )
-        return (datasets_blob_link, streams_blob_link)
```

Comment:
Seems like we actually want either a `yield` statement or to return a list of tuples if the goal is to export all namespaces.